script/server

```diff
@@ -1,8 +0,0 @@
-#!/bin/sh
```

Comment:
Why did you remove this ? 

https://github.com/github/scripts-to-rule-them-all

Reply:
Good spot I didn't migrate them. I will do so soon